src/trumbowyg.js

```diff
@@ -455,7 +456,7 @@ jQuery.trumbowyg = {
             try {
                 // Disable image resize, try-catch for old IE
                 t.doc.execCommand('enableObjectResizing', false, false);
-                t.doc.execCommand('defaultParagraphSeparator', false, 'p');
+                t.doc.execCommand('defaultParagraphSeparator', false, t.o.textInParagraphs ? 'p' : '');
```

Comment:
Honestly, no. Cross browser support, as far as the documentations suggest, is for `div` and `p` only. So i guess this feature is also not for you because `div` tags are no option for trumbowyg, as stated in my other pull request.